pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUser.java

```diff
@@ -16,9 +15,8 @@
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
 
 public interface JdbiUser extends SqlObject {
-
-    @CreateSqlObject
-    JdbiUserStudyEnrollment getJdbiUserStudyEnrollment();
+    @SqlQuery("select guid from user where user_id = :user_id")
```

Comment:
How about having clients just call `findByUserId()` to use the object  form?   Method-per-column-to-return is a bit of an antipattern.